account_manager/src/common.rs

```diff
@@ -19,3 +26,47 @@ pub fn base_wallet_dir(matches: &ArgMatches, arg: &'static str) -> Result<PathBu
         PathBuf::new().join(".lighthouse").join("wallets"),
     )
 }
+
+pub fn read_mnemonic_from_cli(
+    mnemonic_path: Option<PathBuf>,
+    stdin_password: bool,
+) -> Result<Mnemonic, String> {
+    eprintln!("");
+    println!("WARNING: IF YOU HAVE RUN THESE KEYS ON ANOTHER CLIENT, YOU RISK BEING SLASHED.");
```

Comment:
Perhaps we should move this warning into `account_manager/src/validator/recover.rs`? It might be nice to have access to this function without printing that.

Also, how about a warning like this?

`WARNING: KEY RECOVERY CAN LEAD TO DUPLICATING VALIDATORS KEYS, WHICH CAN LEAD TO SLASHING.`